build-tools/packages/build-cli/src/commands/run/codeCoverageStats.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { execSync } from "node:child_process";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { Flags } from "@oclif/core";
+
+import { BaseCommand } from "../../library/index.js";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+export default class RunCodeCoverageStats extends BaseCommand<typeof RunCodeCoverageStats> {
+	static readonly description = "Run comparison of code coverage stats";
+
+	static readonly flags = {
+		dangerfile: Flags.file({
+			description: "Path to dangerfile",
+			required: false,
+		}),
+		...BaseCommand.flags,
+	};
+
+	public async run(): Promise<void> {
+		const scriptPath = path.join(__dirname, "../../library/codeCoverageDangerFile.cjs");
+
+		execSync(`npx danger ci -d ${scriptPath}`, { stdio: "inherit" });
```

Comment:
Does the fact we're using `ci` here affect running the new command locally?

Reply:
No, we can run it locally too. I partially tested it by running locally.